index.html

```diff
@@ -1412,6 +1412,91 @@ <h3>Biometrics</h3>
     </section>
   </section>
 
+  <section class="informative">
+    <h2>Environmental and Ethical Considerations</h2>
+
+    <p>
+      The following section details certain key areas of focus when implementing a 
+      new DID method, or implementing a solution that incorporates DIDs.
+    </p>
+
+    <p>
+      A reliable guide for making assessments of various technologies and weighing
+      ethical considerations is the 
+      <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG Ethical Web Principles</a>
+      document.
+    </p>
+
+    <section class="informative">
+      <h3>Broad Ethical Principles</h3>
+      <p>
+        Decentralized Identifiers may come to underpin much of our digital life. This 
+        may include our public social and career personas, as well our private personas 
+        shared between friends and family. Identifiers representing these personas and 
+        relationships are some of the most important in people's lives, and great care 
+        should be taken when securing an identifier system that supports these critical 
+        activities. As with all things, strong consideration for the appropriate and ethical 
+        use of technology should be made when implementing items related to DIDs.
+      </p>
+
+      <p>
+        As noted in the <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#principles">Principles</a>
+        section of the Ethical Web Principles, there are certain key goals that should 
+        apply to all Web standards and technologies. DIDs explicitly support several
+        of these goals, especially the following:
+        <ul>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy">Privacy</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#control">Individual Control</a></li>
+          <li><a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#multi">Device Independence</a></li>
+        </ul>
+      </p>
+    </section>
+
+    <section class="informative">
+      <h3>Environmental Considerations</h3>
+      <p class="advisement">
+        The following section reflects the views of some members of the working group.
+        Additional PRs are welcome from the working group with additional points of view.
+      </p>
+
+      <section class="informative">
+        <h3>Environmental Principles as Applied to DID method development</h3>
+
+        <p>
+          The guiding principle that 
+          <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
+          web must be an environmentally sustainable platform</a> should be followed.
+          <br />
+          When implementing or utilizing a DID method, consideration should be given
+          to the environmental impacts of any underlying technologies. 
```

Comment:
```suggestion
          DIDs follow the guiding principle that <a 
          href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
          web must be an environmentally sustainable platform</a>.
        </p>
        <p>
          When implementing or utilizing a DID method, also consider the environmental 
          impacts of any underlying technologies.
```

Reply:
I also think the Web should also be fun, collaborative, fast, and helpful, but I am not sure if we can really quantify such subjective things well enough to throw in MUST that doesn't indicate any specific action for the developer reading this guide.

Reply:
@csuwildcat — I've adjusted the language used here. Also, please note that as a NOTE, this document is purely informative and contains no normative statements, even if normative language is used. (You may note that I reworded a number of other previously "must" statements.) In this case, that word was part of a phrase quoted from the linked document, but as it could be confusing, it's not here anymore.

Reply:
Also, please replace "underlying technologies" 
```suggestion
          The guiding principle that 
          <a href="https://www.w3.org/2001/tag/doc/ethical-web-principles/#sustainable">the
          web must be an environmentally sustainable platform</a> should be followed.
          <br />
          When implementing or utilizing a DID method, consideration should be given
          to the environmental impacts caused by the method.
```

Reply:
@jandrieu 
> Also, please replace "underlying technologies"

Reasonable, and done.